test/resources/failure/EMI40_RLC_V4.csv

```diff
@@ -0,0 +1 @@
+2023-09-16,no,,Jim,Jeremy,Smith-Smythe,JK883044D,120/SB09591,25.00,yes,22.8598,
```

Comment:
in which test is this and below used?

Reply:
I used this and its successful counterpart to do manual testing of the service.  It's essentially the same thing I did for ers-checking-frontend.

Reply:
ok, so if it is not used it is not needed

Reply:
Done in 6e500f8.